templates/compose/bluesky-pds.yaml

```
@@ -7,61 +7,47 @@
 
 services:
   pds:
-    image: 'ghcr.io/bluesky-social/pds:latest'
+    image: 'ghcr.io/bluesky-social/pds:0.4.182'
     volumes:
-      - ./pds-data:/pds
+      - './pds-data:/pds'
```

will have to insist making this a volume instead of bind mount, unless there is a specific reason for it. There are often bugs in coolify that cause the permissions for the `/data` directory to be overriden. This will make every app with bind mounts on the root stop working,
```suggestion
      - pds-data:/pds
```